web/src/components/storage/LvmPage.tsx

```diff
@@ -0,0 +1,177 @@
+    setErrors(errors);
+
+    if (errors.length) return;
+
+    addVolumeGroup(
+      name,
+      selectedDevices.map((d) => d.name),
+      moveMountPoints,
+    );
+    navigate(PATHS.root);
+  };
+
+  return (
+    <Page>
+      <Page.Header>
+        <Content component="h2">{_("New Volume Group")}</Content>
+        <SubtleContent>{_("Create a new LVM volume group")}</SubtleContent>
+      </Page.Header>
+
+      <Page.Content>
+        <Form id="lvmForm" onSubmit={onSubmit}>
+          {errors.length > 0 && (
+            <Alert variant="warning" isInline title={_("Something went wrong")}>
+              {errors.map((e, i) => (
+                <p key={`error_${i}`}>{e}</p>
+              ))}
+            </Alert>
+          )}
+          <FormGroup fieldId="lvmName" label={_("Name")} isStack>
+            <TextInput id="lvmName" value={name} onChange={updateName} />
+          </FormGroup>
+          <FormGroup label={_("Disks")} role="group" style={{ justifySelf: "stretch" }} isStack>
+            <Content component="small">
+              {_(
+                "The needed LVM physical volumes will be created as partitions on the chosen disks, based on the sizes of the logical volumes. If you select more than one disk, the physical volumes may be distributed along several disks.",
+              )}
+            </Content>
+            <Gallery hasGutter>
+              {allDevices.map((device) => (
+                <Checkbox
+                  key={device.sid}
+                  id={`device${device.sid}`}
+                  label={<Content isEditorial>{deviceLabel(device)}</Content>}
+                  description={
+                    <Flex rowGap={{ default: "rowGapXs" }} columnGap={{ default: "columnGapSm" }}>
+                      <SubtleContent>
+                        {typeDescription(device)} {contentDescription(device)}
+                      </SubtleContent>
+                      {filesystemLabels(device).map((s, i) => (
+                        <Label key={i} variant="outline" isCompact>
+                          {s}
+                        </Label>
+                      ))}
+                      {device.systems.map((s, i) => (
+                        <Label key={i} isCompact>
+                          {s}
+                        </Label>
+                      ))}
+                    </Flex>
+                  }
+                  isChecked={selectedDevices.includes(device)}
+                  onChange={() => updateSelectedDevices(device)}
+                />
+              ))}
+            </Gallery>
+          </FormGroup>
+          <FormGroup label={_("Move mount points")} isStack>
+            <Checkbox
+              id="moveMountPoints"
+              label={_(
+                "Create logical volumes for the mount points currently configured at the selected disks",
+              )}
+              isChecked={moveMountPoints}
+              onChange={(_, v) => setMoveMountPoints(v)}
```

Comment:
I think it was we who wrote this line, but reviewing it I realize we should actually do not trust in the checkbox value for toggling its state here. Since it is a controlled checkbox, better to go for a `setMoveMountPoints(!moveMountPoints)` or even create the `toggleMountPointConversion = () => setMoveMountPoints(!moveMountPoints)` method.